mqtt3/1.0.0/modules/ROOT/pages/mqtt3-connector-examples.adoc

```diff
@@ -0,0 +1,22 @@
+= MQTT3 Connector Examples - Mule 4
```

Comment:
Is there a reason we have to mention "Mule 4" everywhere? We have already mentioned that MQTT 3 Connector needs Mule 4.

Reply:
Yes, this is to differentiate Mule 3 connectors from Mule 4 connectors.

Reply:
We are not going to build this connector for Mule 4. So I would recommend we call "Mule 4" as a prerequisite and avoid calling it out everywhere. Anything extra in documentation adds complexity for the reader.